ReactiveObjCTests/RACNeverSignalSpec.m

```diff
@@ -41,6 +41,22 @@
   }
```

Comment:
and check similarly the `subscriber` is not deallocated even if the signal was deallocated